--- a/configure
+++ b/configure
@@ -7717,7 +7717,7 @@
 $as_echo "CFLAGS=$foundthrflag and LIBS=$foundthrlib" >&6; }
 { $as_echo "$as_me:$LINENO: checking POSIX threads usability" >&5
 $as_echo_n "checking POSIX threads usability... " >&6; }
-if test "$cross_compiling" = yes; then
+if test yes; then
   { $as_echo "$as_me:$LINENO: result: can't test because we are cross-compiling" >&5
 $as_echo "can't test because we are cross-compiling" >&6; }
 else
