Index: iftop-1.0~pre2/configure
===================================================================
--- iftop-1.0~pre2.orig/configure	2012-07-13 17:42:38.952268457 +0200
+++ iftop-1.0~pre2/configure	2012-07-13 17:43:10.842514450 +0200
@@ -7717,7 +7717,7 @@
 $as_echo "CFLAGS=$foundthrflag and LIBS=$foundthrlib" >&6; }
 { $as_echo "$as_me:$LINENO: checking POSIX threads usability" >&5
 $as_echo_n "checking POSIX threads usability... " >&6; }
-if test "$cross_compiling" = yes; then
+if test yes; then
   { $as_echo "$as_me:$LINENO: result: can't test because we are cross-compiling" >&5
 $as_echo "can't test because we are cross-compiling" >&6; }
 else
@@ -8458,7 +8458,8 @@
 if test -n "$CONFIG_FILES"; then
 
 
-ac_cr=''
+ac_cr='
+'
 ac_cs_awk_cr=`$AWK 'BEGIN { print "a\rb" }' </dev/null 2>/dev/null`
 if test "$ac_cs_awk_cr" = "a${ac_cr}b"; then
   ac_cs_awk_cr='\\r'
