--- a/options.c
+++ b/options.c
@@ -47,6 +47,8 @@
             "dummy",
             "vmnet",
             "wmaster", /* wmaster0 is an internal-use interface for mac80211, a Linux WiFi API. */
+            "phonet",  /* phonet0 is Nokia's Phone Network Protocol for modem interfaces */
+            "upnlink", /* upnlink0 is another Nokia interface */
             NULL       /* last entry must be NULL */
         };
 
