Index: iftop-1.0~pre2/options.c
===================================================================
--- iftop-1.0~pre2.orig/options.c	2012-07-13 17:43:40.022730257 +0200
+++ iftop-1.0~pre2/options.c	2012-07-13 17:44:54.783244441 +0200
@@ -47,6 +47,8 @@
             "dummy",
             "vmnet",
             "wmaster", /* wmaster0 is an internal-use interface for mac80211, a Linux WiFi API. */
+            "phonet",  /* phonet0 is Nokia's Phone Network Protocol for modem interfaces */
+            "upnlink", /* upnlink0 is another Nokia interface */
             NULL       /* last entry must be NULL */
         };
 
