--- iftop-0.17.orig/configure
+++ iftop-0.17/configure
@@ -6348,7 +6348,7 @@
 echo "${ECHO_T}CFLAGS=$foundthrflag and LIBS=$foundthrlib" >&6
 echo "$as_me:$LINENO: checking POSIX threads usability" >&5
 echo $ECHO_N "checking POSIX threads usability... $ECHO_C" >&6
-if test "$cross_compiling" = yes; then
+if test yes; then
   echo "$as_me:$LINENO: result: can't test because we are cross-compiling" >&5
 echo "${ECHO_T}can't test because we are cross-compiling" >&6
 else
