--- iftop-0.17.orig/addrs_ioctl.c
+++ iftop-0.17/addrs_ioctl.c
@@ -132,6 +132,9 @@
         if (strcmp(ifa->ifa_name, interface))
            continue; /* Not our interface. */
 
+        if (ifa->ifa_addr == NULL)
+           continue; /* Skip NULL interface address. */
+
         if ( (ifa->ifa_addr->sa_family != AF_INET)
               && (ifa->ifa_addr->sa_family != AF_INET6) )
            continue; /* AF_PACKET is beyond our scope. */
