Index: iftop-0.17/options.c
===================================================================
--- iftop-0.17.orig/options.c	2011-10-13 08:16:44.245709460 +0200
+++ iftop-0.17/options.c	2011-10-13 08:17:04.265495960 +0200
@@ -46,6 +46,7 @@
             "gif",     /* psuedo-device generic tunnel interface */
             "dummy",
             "vmnet",
+            "wmaster", /* wmaster0 is an internal-use interface for mac80211, a Linux WiFi API. */
             NULL        /* last entry must be NULL */
         };
 
